x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/index.tsx

```diff
@@ -207,7 +213,47 @@ export const AlertIndex = memo(() => {
     [setVisibleColumns, visibleColumns]
   );
 
-  const selectedAlertData = useAlertListSelector(selectors.selectedAlertDetailsData);
+  useEffect(() => {
```

Comment:
Does this logic need to be active everywhere in the app?

Reply:
i would imagine that it only needs to be active on the alerts page at the moment unless you can think of any other situations?